examples/translation-job-items.json

```diff
@@ -0,0 +1,32 @@
+[
```

Comment:
There will be no such listing, but in our brainstorming you told to write down the "in theory" possible types.

It is called translation-job-items, because translation-item-types would not have an `id`a `field`and a `title` (`name`).